network/base/views.py

```diff
@@ -116,21 +118,36 @@ def settings_site(request):
     return render(request, 'base/settings_site.html')
 
 
-def observations_list(request):
-    """View to render Observations page."""
-    observations = Observation.objects.order_by('-id')[:20]
-    satellites = Satellite.objects.all()
-
-    if request.method == 'GET':
-        form = SatelliteFilterForm(request.GET)
-        if form.is_valid():
-            norad = form.cleaned_data['norad']
-            observations = Observation.objects.filter(satellite__norad_cat_id=norad)
-            return render(request, 'base/observations.html', {'observations': observations,
-                          'satellites': satellites, 'norad': int(norad)})
-
-    return render(request, 'base/observations.html',
-                  {'observations': observations, 'satellites': satellites})
+class ObservationListView(ListView):
+    """
+    Displays a list of observations with pagination
+    """
+    model = Observation
+    ordering = '-id'
+    context_object_name = "observations"
+    paginate_by = 25
```

Comment:
I'd prefer we move this value to settings, instead of hardcoding it here. Something like `ITEMS_PER_PAGE`.